1.0/index.md

```diff
@@ -5,7 +5,7 @@ title: Welcome
 
 <section id="future" class="main-bg">
   <div class="panel left">
-    <img src="./images/polymer1.0-01.svg">
+    <img src="./images/polymer1.0-01.svg" alt="">
```

Comment:
This is intentionally blank alt text, because the image is decorative and doesn't add anything.

If we add the alt text, the screen reader just says, "Image: Polymer 1.0 logo".